shared/js/l10n.js

```diff
@@ -1662,20 +1662,25 @@
       }
     }
 
+    var additionalLanguagesPromise = Promise.resolve();
```

Comment:
it would make sense to initialize this as null, and create Promise.resolve() in "else" of the if below so that we don't create the Promise for most common pathway (and BTO).